driver/pom.xml

```diff
@@ -109,10 +109,12 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
             </plugin>
+<!--
```

Comment:
We cna remove this?  If this is to disable it I suggest adding it to top level and use the checkstyle property to disable it